latest_cov_build.py

```diff
@@ -59,19 +63,29 @@
 if not os.path.exists('tools'):
     os.makedirs('tools')
 
-# Download
-name = os.path.join('tools', version)
+# Download geckodriver
+name_gecko = os.path.join('tools', version)
 base_url += version
-urlretrieve(base_url, name)
+urlretrieve(base_url, name_gecko)
+
+# Download grcov
+name_grcov = os.path.join('tools', 'grcov')
+urlretrieve(grcov_url, name_grcov)
 
-# Extract
+# Extracting archive for geckodriver
 if version.endswith('zip'):
-    with zipfile.ZipFile(name, 'r') as zip_ref:
+    with zipfile.ZipFile(name_gecko, 'r') as zip_ref:
         zip_ref.extractall('tools')
 
 # MacOS and Linux OS have 'tar.gz' extensions
 elif base_url.endswith('tar.gz'):
-    with tarfile.open(name, 'r:gz') as tar:
+    with tarfile.open(name_gecko, 'r:gz') as tar:
         tar.extractall(path='tools')
 
-os.remove(name)
+os.remove(name_gecko)
+
+# bzip archives
+for filename in [name_grcov, 'tools/target.tar.bz2']:
```

Comment:
Put all the extraction code in the for loop, using the correct code according to the extension.

Reply:
Should I add `.zip` into the loop too? 
I thought that because geckodriver can be either `.zip` or `.tar.gz` it would be more elegant to extract it separately.

Reply:
Yes, let's put everything here, this way we don't have to do anything specific for any of the things we download.